Dockerfile

```diff
@@ -0,0 +1,7 @@
+ARG RUNTIME=python3.8
```

Comment:
@vagelim , a limitation of this action is that the python runtime is hardcoded to a python version (python3.8). I added  ARG RUNTIME to the Dockerfile, but, there is no way to provide docker --build-args to the Docker action. The only parameters we can pass are fed into the entrypoint, so, we can’t dynamically choose the tag of the Docker image to use.

Reply:
makes sense. we could do hacky ways around this but for now we can live with the restriction and in time they may update the capabilities